internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -29,6 +29,12 @@ const (
 	disabled = "DISABLED"
 )
 
+// SQS Queue configs
```

Comment:
nit: 
```suggestion
// SQS Queue field values.
```
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences